scaffolding-chef-inspec/lib/scaffolding.ps1

```diff
@@ -24,7 +24,12 @@ function Invoke-DefaultBefore {
     }
 
     if((Select-String -Path "$PLAN_CONTEXT/../inspec.yml" -Pattern "compliance: ")){
-        Invoke-ComplianceLogin
+        if(!$(Test-Path "~/.inspec/compliance/config.json")){
+            inspec compliance login "$scaffolding_automate_server_url" `
+                            --insecure `
```

Comment:
Is it strictly necessary to use `--insecure` ? I'd like to avoid it if possible.